artillery/artillery-data.js

```diff
@@ -0,0 +1,14 @@
+const generateRandomId = (userContext, events, done) => {
+  const min = 5000000;
```

Comment:
Project summary wants us to test using mostly IDs from last 10% of our datasets. Consider a min of 9,000,000.